tests/msc2716_test.go

```diff
@@ -1147,14 +1154,38 @@ func batchSendHistoricalMessages(
 	return res
 }
 
-func getEventsFromBatchSendResponseBody(t *testing.T, body []byte) (eventIDs []string) {
-	eventIDs = client.GetJSONFieldStringArray(t, body, "events")
+func getStateEventIDsFromBatchSendResponseBody(t *testing.T, body []byte) (stateEventIDs []string) {
+	stateEventIDs = client.GetJSONFieldStringArray(t, body, "state_event_ids")
+
+	return stateEventIDs
+}
+
+func getEventIDsFromBatchSendResponseBody(t *testing.T, body []byte) (eventIDs []string) {
+	eventIDs = client.GetJSONFieldStringArray(t, body, "event_ids")
 
 	return eventIDs
 }
 
-func getNextChunkIdFromBatchSendResponseBody(t *testing.T, body []byte) (nextChunkID string) {
+func getNextChunkIDFromBatchSendResponseBody(t *testing.T, body []byte) (nextChunkID string) {
 	nextChunkID = client.GetJSONFieldStr(t, body, "next_chunk_id")
 
 	return nextChunkID
 }
+
+func getInsertionEventIDFromBatchSendResponseBody(t *testing.T, body []byte) (insertionEventID string) {
+	insertionEventID = client.GetJSONFieldStr(t, body, "insertion_event_id")
+
+	return insertionEventID
+}
+
+func getChunkEventIdFromBatchSendResponseBody(t *testing.T, body []byte) (chunkEventId string) {
+	chunkEventId = client.GetJSONFieldStr(t, body, "chunk_event_id")
```

Comment:
I don't understand the rationale for splitting this out to a different function. `getChunkEventIdFromBatchSendResponseBody` is a convolutedly named proxy function for something which is clearer (and in fact shorter!): `client.GetJSONFieldStr(t, body, "chunk_event_id")`

This is just extra code which makes it less clear to see what operations are actually happening in the tests. Am I missing something here?

Reply:
I'm not sure anymore. Probably just morphed from something more involved and now useless. Removed ⏩